.pre-commit-config.yaml

```diff
@@ -9,9 +9,11 @@ repos:
         stages: [commit-msg]
```

Comment:
please take a moment to remove the commented-out entries in this file. they are no longer necessary